kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.testing.collection;
+
+import java.util.ArrayList;
+
+public class OddNumbersExterminator {
+
+    ArrayList<Integer> evenNumbers = new ArrayList<>();
```

Comment:
trzymanie evenNumbers jako zmienna klasowa nie jest dobrym pomyslem. Jesli wywolasz 2 x metode exterminate, to evenNumbers bedzie zawieralo liczby parzyste z obu wywolan co raczej jest niepozadane (chcesz dostac parzyste z tylko ostatniego wywolania). 

Niech metoda extermianate zwraca nowa liste, tylko z liczbami parzystymi